medium-guide/readme.md

```diff
@@ -4,32 +4,28 @@
 
 # How to publish to Medium
 
-At New Atoms we write on our [New Atoms Medium Blog channel](https://medium.com/new-atoms-beta). We write about what we learn and what it's like to start a business. Every team member can pitch ideas for blogposts in the Slack channel: beta.
+At New Atoms we write on our [New Atoms Medium Blog channel](https://medium.com/new-atoms-beta). We write about what we learn and what it's like to start a business. Every team member can pitch ideas for blogposts in the [Slack](../slack-guide/) channel: beta.
 
 This guide is a quick tutorial about how to create a blogpost.
 
-> **Frequent dispatches from the front of journalistic experimentation.**
-
->New Atoms is an early-stage news, journalism and research organisation with collaboration — both between humans, and humans and computers — at its core.
-
 Signup on [Medium](https://medium.com/) and ask a team member to invite you as an editor.
 
 ## Write and publish
 
-Click on your profile pick and select new story. You can now write your blogpost add images and make up your text. The [Medium Help center](https://help.medium.com/hc/en-us) learns you a lot about how you can make things work.
+Click on your profile picture and select **new story**. You can now write your blogpost, add images and style your text. The [Medium Help center](https://help.medium.com/hc/en-us) is a great resource for getting started and finding help if you get stuck.
 
 To publish a story on the New Atoms Channel, do the following:
 
-If you've written your awesome post, choose add to publication an then tick the New Atoms channel. Add draft to publication, now others can review your post because its in draft & stories.
+If you've written your awesome post, choose add to publication and then click **New Atoms channel**. Once you click on **add draft to publication** others can review your post in draft & stories.
 
 1.<img src="../images/medium-add-to-publication.png" width="200"> 2.  <img src="../images/medium-add-draft.png" width="200">
 
 ## Edit
 
-If you're asked to review or edit a medium blogpost always comment and not change. The writer should choose to implement comments. When commenting on text just highlight the text where you want to comment on. Click the comment icon <img src="../images/medium-review-icon.png" width="20"> and send.
+If you're asked to review or edit a Medium blogpost always comment -- never change it yourself. The writer should always choose to implement suggestions. When commenting on text just highlight the text where you want to comment on. Click the comment icon <img src="../images/medium-review-icon.png" width="20"> and send.
```

Comment:
Use a em-dash in place of `--`, also it shouldnt be padded by spaces.

The way you make an `—` is by doing <kbd>alt</kbd> + <kbd>shift</kbd> + <kbd>-</kbd>